lesson7-questions.txt

```diff
@@ -87,19 +117,39 @@ You will see the SQL that is executed logged to the screen.  What is that SQL?
 There is a security exposure, a risk that a hacker could do damage.  What is that
 risk? What two things are done in your code to prevent the risk?
 
+The risk is that unauthorized user can delete the post.
+
+One thing that prevennt the risk is:
+  before_action :check_logon, except: %w[show] 
+Which checks if the user is logged in before any action except showing the forum content
+
+One more thing is:
```

Comment:
You would also want to add a before_action :authenticate_user! to make sure the user is logged in and owns the post (@post.user == current_user).